### PR TITLE
DEMO #3 Fix

### PR DESCRIPTION
**Pull request by Axedez**
[Trello Task](https://trello.com/_____)

**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [ ] screenshot of feature is added if task has UI

### CODE REVIEW

**Comment by Axedez** [comment]
test

**Review by Ukhanskyi — APPROVED** [review]

File: app/models/news.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class News < ApplicationRecord
 
   scope :visible, -> { where(is_visible: true) }
   scope :ordered, -> { order('created_at DESC') }
+  scope :osbb_by_user, ->(user_id) { where(osbb_id: user_id.osbb_id) }
```
Comment (pereslop) [inline app/models/news.rb:10]: in controller you pass current_user as an argument, `News.osbb_by_user(current_user).visible`, so better to name argument user, instead `user_id`

File: app/javascript/components/users/show/_InfoHelper.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const InfoHelper = ({ name, value, link}) => {
```
Comment (pereslop) [inline app/javascript/components/users/show/_InfoHelper.jsx:4]: extra space after bracket

File: app/javascript/components/users/show/_InfoHelper.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <>
+    <div className='user-item'>
+      <span className='naming-user'>{ name }</span>
+      <span className='insert-user'>{ value? value : 'Unknown'  } { link }</span>
```
Comment (pereslop) [inline app/javascript/components/users/show/_InfoHelper.jsx:9]: `value? value : 'Unknown'` is it ternary operator?, if yes needs space before question mark

**Comment by chu4** [comment]
add test for 404 responses

**Review by matsevich — APPROVED** [review]

**Review by chu4 — APPROVED** [review]

**Pull request merged**